### PR TITLE
Core test

### PR DESCRIPTION
**Pull request by gearama**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

File: sdk/core/perf/test/CMakeLists.txt [hunk 1]
```diff
@@ -38,8 +38,16 @@ target_include_directories(
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
 )
 
+if(DEFINED ENV{VCPKG-AZURE-CORE-CPP})
```
Comment (LarryOsterman) [inline sdk/core/perf/test/CMakeLists.txt:41]: I take it this gets set by the pipeline yml file?

Are there circumstances where you would expect this to be set by SDK authors?

**Review by LarryOsterman — APPROVED** [review]

File: sdk/core/perf/test/CMakeLists.txt (hunk 1 above)
Comment (gearama) [reply sdk/core/perf/test/CMakeLists.txt:41]: nope

**Pull request merged**